app/presenters/notifications/setup/event.presenter.js

```diff
@@ -0,0 +1,106 @@
+'use strict'
+
+/**
+ * Formats the session data to create an event for 'wabs.public.events'
```

Comment:
For reference, `wabs` is just what we've called the DB in our local environment. It's different in all of them so `wabs` might not be understood by those that follow.

Also, `public.events` is actually a view of the underlying table `water.events`, which is where the record will get persisted.

So, you might be better off referring to the model, and each model has a comment that references the underlying view and table.

 ```suggestion
 * Formats a notification `SessionModel` instance into the data needed for a 'EventModel' record
```

Reply:
done